tm.plugin.lexisnexis/R/readLexisNexisHTML.R

```diff
@@ -56,7 +56,13 @@ readLexisNexisHTML <- FunctionGenerator(function(elem, language, id) {
         names(nodes) <- sapply(nodes, function(x) xml_text(xml_children(xml_children(x)[[1]])[1], trim=TRUE))
         vals <- sapply(nodes, xml_text, trim=TRUE)
 
-        copyright <- vals[[max(which(grepl("^Copyright", vals)))]]
+        cr <- which(grepl("^Copyright", vals))
+        if (any(cr)) {
+            copyright <- vals[[max(cr)]]
+        } else {
+            warning(sprintf("Could not parse copyright notice for article %s. This may indicate a problem with the source data, as LexisNexis copyright notices are nearly universal.\n", id))
```

Comment:
Does it really indicate a problem with the source? In your case, do you have any idea what produced the problem?

Reply:
My dataset has 48 unparseable copyright notices, of which 47 are instances of an 'error' document like the one in the PR comment (at last once from six of my 23 newspapers), and one is a Washington Post article which is otherwise complete and valid but has a "Copy 2013 Washington Post ..." copyright notice rather than "Copyright 2013 Washington Post ...", so it doesn't trigger the regex.

So I suspect that most examples of this code being triggered will be non-English users hitting error message. But it's possible there are a few other malformed documents like the WaPo one.